.release-please-manifest.json

```diff
@@ -1,16 +1,16 @@
 {
-  "packages/api-explorer": "21.0.10",
-  "packages/extension-api-explorer": "21.0.10",
-  "packages/extension-sdk": "21.0.10",
-  "packages/extension-sdk-react": "21.0.10",
-  "packages/hackathon": "21.0.10",
-  "packages/run-it": "21.0.10",
-  "packages/sdk": "21.0.10",
-  "packages/sdk-codegen": "21.0.10",
-  "packages/sdk-codegen-scripts": "21.0.10",
+  "packages/api-explorer": "21.1.0",
+  "packages/extension-api-explorer": "21.4.0",
+  "packages/extension-sdk": "21.4.0",
+  "packages/extension-sdk-react": "21.4.0",
+  "packages/hackathon": "21.4.0",
+  "packages/run-it": "21.1.0",
+  "packages/sdk": "21.4.0",
+  "packages/sdk-codegen": "21.0.11",
+  "packages/sdk-codegen-scripts": "21.0.11",
   "packages/sdk-codegen-utils": "21.0.10",
-  "packages/sdk-node": "21.0.10",
+  "packages/sdk-node": "21.4.0",
   "packages/sdk-rtl": "21.0.10",
```

Comment:
should we take this opportunity to rename and reset versioning on
- `@looker/sdk-codegen`
- `@looker/sdk-codegen-scripts`
- `@looker/sdk-codegen-utils`
- `@looker/sdk-rtl`

That should be a separate PR that we merge and re-generate this release PR afterwards. I can do that work if someone can provide me what the new names (just removing `sdk-` prefix?) and initial release versions should be.

Reply:
an example start for this work (sdk-rtl only): https://github.com/looker-open-source/sdk-codegen/pull/534

Reply:
gonna shelve this topic for now